recipe-server/client/control/components/Fields.js

```diff
@@ -50,52 +50,68 @@ buildControlField.propTypes = {
   children: pt.node,
 };
 
-export const CheckboxGroup = ({ name, onChange, options = [], value }) => {
+export class CheckboxGroup extends React.Component {
+  static propTypes = {
+    name: pt.string.isRequired,
+    onChange: pt.func.isRequired,
+    options: pt.array,
+    value: pt.arrayOf(pt.string),
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
We follow this convention throughout the rest of the front end - _if_ we use a constructor on a `React.Component` we define the state regardless if it's used or not.

If you'd prefer, this can be removed across the front end.